### DISCUSSION TITLE
Rerender

### DISCUSSION THREAD

**Discussion by joseserrano15** [l-lin/angular-datatables#1589] [Q&A] [answered]
Hello. Does anyone know how can reload the table but keep in the current page. Actually, if I am in the last page and I reload the table it is moving to the first page. I'm using the example of rerender: https://l-lin.github.io/angular-datatables/#/advanced/rerender
Thanks

**Comment by shanmukhateja** [comment, marked as answer]
Hi,

In your case you need `dtInstance.ajax.reload(false)` instead of `this.dtTrigger.next()`. Also remove `dtInstance.destroy()` call.

Documentation: [ajax.reload](https://datatables.net/reference/api/ajax.reload())

The `false` parameter is necessary in your case to inform DT to NOT go back to the first page.

**Reply by joseserrano15** [reply]
Thanks, it worked perfect.
